### DISCUSSION TITLE
Reset is Not a Function and Unexpected Token

### DISCUSSION THREAD

**Discussion by webdevNYC** [react-hook-form/react-hook-form#6049] [General]
We are trying to use the code in your docs for Reset: https://react-hook-form.com/api/useform/reset
We have copied the code exactly. If we just use the regular `<input type="reset" />`  then the form resets, but the problem is that the form doesn't reset to the default values, and keeps the state. So we require the other options. For those you have
have the code as follows: `<input type="button" onClick={() => reset({ firstName: "bill" }); }} /> `
or
```
<input type="button" onClick={() => {
        reset({
          firstName: "bill"
        }, {
          keepErrors: true, 
          keepDirty: true,
          keepIsSubmitted: false,
          keepTouched: false,
          keepIsValid: false,
          keepSubmitCount: false,
        });
      }} />
```

2 problems:
1. Using the first input code  causes an error: Unexpected Token.
2. Using the 2nd code, causes an error: Reset is not a function

How do we actually implement Reset, if it's not a function? How do we pass the values to it?

**Comment by bluebill1049** [comment]
did you pass `reset` down as prop? and try to console.log that `reset` variable. this is most likely due to missing prop.
